src/services/CityConfigService.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import type {Config} from '../Config'
+import format from 'string-format'
+import CityConfig from '../cities/CityConfig'
+import cityConfigs from '../cities/cityConfigs'
+
+export default class CityConfigService {
+  config: Config
+
+  constructor (config: Config) {
+    this.config = config
+  }
+
+  getFilledCityConfigs (): Array<CityConfig> {
+    const baseUrl = `${this.config.host}:${this.config.port}`
+    const cityConfigsArray: Array<CityConfig> = Object.values(cityConfigs)
```

Comment:
This currently produces a flow error because `Object.values()` returns an `Array<mixed>`. The issue is known since two years ago (see [here](https://github.com/facebook/flow/issues/2221)). 
Any ideas what we should do about this @Schedulaar @maxammann? Not using `Object.values()` can't be the solution here.

Reply:
The `cityConfig` object is really practical in the backend, we can easily get the CityConfig for a specific city without having to search all configs. Also I don't like that I would have to change code just because flow does not support it :)
I will add the flow exception.